client/src/island/hexalot.ts

```diff
@@ -69,34 +77,60 @@ export class Hexalot {
         }
     }
 
-    public load(): void {
-        this.refreshFingerprint()
-        const genomeData = this.appStorage.getGenomeData(this)
-        if (genomeData) {
-            this.genome = fromGenomeData(genomeData)
-        }
-        this.rotation = this.appStorage.getRotation(this)
-    }
-
     public get id(): string {
         if (!this.identifier) {
             throw new Error("Should have refreshed fingerprint first")
         }
         return this.identifier
     }
 
-    public get occupied(): boolean {
-        return !!this.genome
-    }
-
     public get isLegal(): boolean {
         return this.spots.every(spot => spot.isLegal)
     }
 
-    public refreshFingerprint(): void {
+    public refreshId(): void {
         this.identifier = spotsToHexFingerprint(this.spots)
```

Comment:
Change the name of `spotsToHexFingerprint` to ID as well

Reply:
did it.